CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.16.1 - 2019-10-16
+### Fixed
+- [Bugs fixed](https://github.com/microsoft/vscode-azureappservice/issues?q=is%3Aissue+is%3Aclosed+milestone%3A0.16.1)
```

Comment:
Should I explicitly list the bugs fixed here or is this alright?

Reply:
I think it's worth listing the bugs here since there are only 2 and they're the sole purpose of this release.